### DISCUSSION TITLE
Javascript Policy

### DISCUSSION THREAD

**Discussion by amitngm** [keycloak/keycloak#23389] [Q&A]
Hi All,

I need to implement attribute based conditions  using java script based policy. But I am unable to see this option in policy type. How can we enable this option in Keycloak 22.0.1.

Regards,
Amit

**Comment by BlackVoid** [comment]
In newer versions of keycloak you need you create a jar file containing the scripts and add that to the providers folder in your keycloak instance. You can find the documentation on how to do it here: https://www.keycloak.org/docs/22.0.3/server_development/#_script_providers

**Comment by amitngm** [comment]
I need to create a policy based on user attribute and resource attribute
and/or contextual attribute.How can I acheive it without js policy?

Thanks in advance

Amit
